ext/sockets/php_sockets.h

```diff
@@ -183,10 +183,15 @@ PHP_SOCKETS_API int socket_import_file_descriptor(PHP_SOCKET socket, php_socket
 #define PHP_SOCKET_EINVAL EINVAL
 #endif
 
+#ifdef WIN32
```

Comment:
what is your intent here ?

Reply:
Keep the format consistent, otherwise I can remove this line if you like?

Reply:
if there is no case for windows (unless @cmb69 says otherwise), seems pointless ; at worst this definition can be put around line 156 or line 406 then.

Reply:
Thanks for the review, removed now.

Reply:
nice changes ;-)